hcl/writer.go

```diff
@@ -465,9 +465,21 @@ func writeTuple(pbody, body *hclwrite.Body, attr string, value cty.Value) {
 }
 
 // Interpolate replaces the hardcoded resources link
-// with TF interpolation
+// with TF interpolation.
+// I has the following format:
+// * VALUE => ${aws_instance.name.attribute}
+// So then each value matching the VALUE will
+// be replaced with the reference
 func (w *Writer) Interpolate(i map[string]string) {
-	if !w.opts.Interpolate {
+	// If Interpolation is disabled or
```

Comment:
I'm sorry to ask silly questions, but how/who decides if "interpolation" is disabled? Wouldn't it be the code detecting it's not needed?

Reply:
We have an `--interpolate` flag, by default is `true`.

Reply:
Shouldn't one of the test be with `Interpolate false` ?